content/docs/gto/user-guide/downstream.md

```diff
@@ -0,0 +1,40 @@
+# Acting downstream
+
+Once Git tag was pushed, you can start acting downstream. One of the popular
+options here is to use CI/CD, setting it to be triggered by pushing a Git tag.
+For general details, check out something like
+[GitHub Actions](https://github.com/features/actions),
+[GitLab CI/CD](https://docs.gitlab.com/ee/ci/) or
+[Circle CI](https://circleci.com).
+
+The other option is to
+[configure webhooks](https://docs.github.com/en/rest/webhooks) that will send
+HTTP requests to your server upon pushing Git tags to the remote.
+
+Finally, you can configure your server to query your Git provider via something
+like REST API to check if changes happened. As an example, check out
+[Github REST API](https://docs.github.com/en/rest).
+
+## Getting started with CI/CD
+
+To act upon registrations and assignments (Git tags), you can create simple CI
+workflow. Check out
+[the example workflow in `example-gto` repo](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
+The workflow uses [the GTO GH Action](https://github.com/iterative/gto-action)
+that fetches all Git tags (to correctly interpret the Registry) and finds out
+the version of the artifact that was registered, or the stage that was assigned,
+so you could use them in later steps of the CI.
```

Comment:
Perhaps wanna add a link to GitLab corresponding? https://docs.gitlab.com/ee/ci/pipelines/